pkg/app/mssql.go

```diff
@@ -0,0 +1,366 @@
+package app
+
+import (
+	"context"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	crv1alpha1 "github.com/kanisterio/kanister/pkg/apis/cr/v1alpha1"
+	"github.com/kanisterio/kanister/pkg/field"
+	"github.com/kanisterio/kanister/pkg/kube"
+	"github.com/kanisterio/kanister/pkg/log"
+)
+
+const (
+	mssqlWaitTimeout = 2 * time.Minute
+)
+
+type MssqlDB struct {
+	cli        kubernetes.Interface
+	namespace  string
+	name       string
+	deployment *appsv1.Deployment
+	service    *v1.Service
+	pvc        *v1.PersistentVolumeClaim
+	secret     *v1.Secret
+	dbUname    string
+	dbPass     string
+}
+
+func NewMssqlDB(name string) App {
+	return &MssqlDB{
+		name: name,
+		// These values are hard coded while creating blueprint it self
+		dbUname: "sa",
+		dbPass:  "MyC0m9l&xP@ssw0rd",
+	}
+}
+
+func (m *MssqlDB) ConfigMaps() map[string]crv1alpha1.ObjectReference {
+	return nil
+}
+
+func (m *MssqlDB) Secrets() map[string]crv1alpha1.ObjectReference {
+	return map[string]crv1alpha1.ObjectReference{
+		"mssql": {
+			Kind:      "Secret",
+			Name:      "mssql",
+			Namespace: m.namespace,
+		},
+	}
+}
+
+func (m *MssqlDB) Init(ctx context.Context) error {
+	cfg, err := kube.LoadConfig()
+	if err != nil {
+		return err
+	}
+
+	m.cli, err = kubernetes.NewForConfig(cfg)
+	if err != nil {
+		return err
+	}
+	return err
+}
+
+func (m *MssqlDB) Install(ctx context.Context, namespace string) error {
+	m.namespace = namespace
+	//Create Secret
+	secret, err := m.cli.CoreV1().Secrets(namespace).Create(ctx, m.getSecretObj(), metav1.CreateOptions{})
+	if err != nil {
+		return err
+	}
+	log.Print("Secret with name " + secret.Name + " created successfully")
+	m.secret = secret
+	// Create PVC
+	pvcObj, err := m.getPVCObj()
+	if err != nil {
+		return err
+	}
+	pvc, err := m.cli.CoreV1().PersistentVolumeClaims(namespace).Create(ctx, pvcObj, metav1.CreateOptions{})
+	if err != nil {
+		return err
+	}
+	log.Print("PVC with name " + pvc.Name + " created successfully")
+	m.pvc = pvc
+
+	// Create Deployment
+	deploymentObj, err := m.getDeploymentObj()
+	if err != nil {
+		return err
+	}
+	deployment, err := m.cli.AppsV1().Deployments(namespace).Create(ctx, deploymentObj, metav1.CreateOptions{})
+	if err != nil {
+		return err
+	}
+	log.Print("Deployment with name " + deployment.Name + " created successfully")
```

Comment:
Just logging `Deployment with name xyz was created successfully` is not that useful. If we look at this message while running integration test we would not know which application this log message is for.